scripts/psiphon_install.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+PSIPHON_PATH=$HOME
+PSIPHON_PYCLIENT_PATH=$PSIPHON_PATH/psiphon-circumvention-system/pyclient
+
+sudo apt-get install mercurial
```

Comment:
This is very debian specific. I suggest, instead, to run the script through `bash -e`, document that mercurial is needed, and then let the script bail out if `hg` is not found.